### ISSUE TITLE
"Processing" message gets stuck when modifying the client selector reactively

### ISSUE THREAD

**Issue by thebarty** [aldeed/meteor-tabular#252]
Hi guys,

I like the new processing indicator, but in my app it does not function stable. 

In a lot of cases it shows up and **does NOT disappear - although the data is loaded.**

Is anyone else having this issue?

I will work on an example to reproduce this as I am using a lot of stuff that might be conflicting, p.e. SubsManager...

**Comment by aldeed** [comment]
I have not seen this yet, but it could be true. Still a work in progress due to the nonstandard way this package loads data.

**Comment by aldeed** [comment]
As a workaround, you could use CSS to give it 0 width and height if it is distracting.

**Comment by thebarty** [comment]
Oh yeah  - thats a great idea. I'll try to reproduce this when I have time. It seems to happen when modiying the selector passed to tabular.

**Comment by petrometro** [comment]
+1 : i have exactly this problem - when i modify the selector reactively, it loads the correct data but 'processing' message sticks

**Comment by rolivav** [comment]
+1 to this problem. Its really annoying.

**Comment by rolivav** [comment]
As a work around, I hid it with:
```
div.dataTables_wrapper div.dataTables_processing {
	visibility: hidden;
}
```
Although I would preffer not to render it at all. width and heigth to 0 doesn't wrap text for some reason.

**Comment by aldeed** [comment]
Does this happen for large collections (probably 100k+) and only when searching? I have seen it get stuck for large collections after entering a search term because it is seemingly taking a long time to do the count. So the docs appear but the count at the bottom would still probably say 0. After awhile (could be 20 or 30 seconds), the processing message disappears and the count is correct.

**Comment by crapthings** [comment]
it happens after you typing fast in filter.

**Comment by rolivav** [comment]
It happens after typing anything in filter, sometimes it will just go after displaying results, sometimes it will stay there forever.

**Comment by thebarty** [comment]
Yeah for me it also happens when switchin a value of a combobox which then generates a modifier and passes it to tabular

**Comment by rsercano** [comment]
+1, I'm having same issue when changing Show ... entries combobox...

**Comment by cormip** [comment]
+1 I erratically see this issue when filtering or sorting (clicking header) on a simple table with only two records. It seems to behave like a race condition problem.

**Comment by chemuto** [comment]
+1 Having this issue for a really small collection (< 10 items). It is quite random

**Comment by mathewwkl** [comment]
+1 Do you guys have any workaround for this issue? How to check the tabular table really finished loading?

**Comment by soitech** [comment]
+1 Have this issue often/randomly on very small collections, not sure what's triggering it as it just stays there after loading the table data. I'm not modifying the selector, searching, or filtering.

**Comment by blabadi** [comment]
+1 It happens randomly whenever the table is refreshing, not always but often enough to notice it like half of the times I interact with the table

**Comment by hashcutdev** [comment]
+1 happens randomly, sometimes when a new page loads, sometimes when the selector is changed reactively. Sometimes a page reload will clear it but sometimes not. FWIW, my table has <10 items so I don't think the database is slow. Also, the proper rows do appear, but the progress indicator never disappears.

**Comment by diapple** [comment]
+1，in the columns data,I use a function instead of property name,and the Processing just dont disappear after refresh the window,and disapper after do some search or sort

**Comment by adrianciurdarean** [comment]
+1: happens when changing selector, even for small collections (less than 20 documents).

**Comment by rudza** [comment]
> +1: happens when changing selector, even for small collections (less than 20 documents).

Same here.

**Comment by aldeed** [comment]
I think this is fixed in 2.0
